src/pytest_allure_spec_coverage/plugin.py

```diff
@@ -9,4 +9,194 @@
+
+CollectorsMapping = MutableMapping[str, Type[Collector]]
+
+
+def safe_get_marker(item: Item, name: str) -> Mark:
+    """Safely get pytest.Item marker"""
+
+    stub_marker = Mark(name="stub", args=(), kwargs={})
+    return item.get_closest_marker(name, stub_marker)
+
+
+def scenario_ids(item: Item) -> Iterable[str]:
+    """Get scenario identifiers from pytest.Item"""
+
+    return safe_get_marker(item, ScenariosMatcher.MARKER_NAME).args
+
+
+def allure_labels(scenario: Scenario) -> List[Label]:
+    """Make labels for Allure from the given scenario"""
+
+    labels = ("parentSuite", "suite", "subSuite")
+    values = [p.display_name for p in scenario.parents]
+    if len(values) > 3:
+        itv = slice(2, None)
+        values[itv] = [".".join(values[itv])]
+    return [Label(label, value) for label, value in zip(labels, values)]
+
+
+def allure_links(scenario: Scenario) -> List[Link]:
+    """Make links for Allure from the given scenario"""
+
+    return [Link(url=scenario.link, name="Scenario", type=LinkType.LINK)]
+
+
+@dataclass(eq=False)
+class ScenariosMatcher:
+    """Match collected test cases with collected scenarios and report missed ones"""
+
+    PLUGIN_NAME: ClassVar[str] = "smatcher"
+    MARKER_NAME: ClassVar[str] = "scenario"
+
+    config: Config
+    collector: Collector
+    reporter: AllureReporter
+
+    scenarios: Collection[Scenario] = field(default_factory=list)
+    matches: Mapping[Scenario, List[pytest.Item]] = field(default_factory=dict)
+
+    @property
+    def missed(self) -> Iterable[Scenario]:
+        """Not implemented scenarios"""
+
+        return (scenario for scenario, items in self.matches.items() if not items)
+
+    def match(self, items: List[pytest.Item]) -> None:
+        """Match collected tests items with its scenarios"""
+
+        self.matches = {sc: [] for sc in self.scenarios}
+        sc_lookup = {sc.id: sc for sc in self.scenarios}
+        for item in items:
+            for key in scenario_ids(item):
+                with suppress(KeyError):
+                    self.matches[sc_lookup[key]].append(item)
+
+    def mark(self) -> None:
+        """Add markers with links to spec for matched items"""
+
+        link_marker = getattr(pytest.mark, ALLURE_LINK_MARK)
+        for scenario, items in self.matches.items():
+            for item in items:
+                item.add_marker(
+                    link_marker(
+                        scenario.link,
+                        name="Scenario",
+                        link_type=LinkType.LINK,
+                    )
+                )
+
+    def report(self) -> None:
+        """Report about not implemented scenarios"""
+
+        for scenario in self.missed:
+            self._report_scenario(scenario)
+
+    def _report_scenario(self, scenario: Scenario) -> None:
+        fake_uuid = uuid4()
+        fake_result = TestResult(
+            uuid=fake_uuid,
+            name=scenario.name,
+            description=scenario.display_name,
```

Comment:
```suggestion
            name=scenario.display_name,
```
`display name` is not a description

Reply:
Removed.